changelog/22.0/22.0.0/summary.md

```diff
@@ -54,6 +56,16 @@ $ vtctldclient ApplySchema --ddl-strategy="gh-ost" ...
 $ vtctldclient ApplySchema --ddl-strategy="pt-osc" ...
 ```
 
+### <a id="vtgate-metrics"/>VTGate Metrics
+
+Added two new metrics for queries with query type, plan type and tablet type as dimension.
+1. QueryProcessed - This counts the number of query executed.
```

Comment:
```suggestion
1. QueryProcessed - This counts the number of queries processed by this vtgate.
```

Reply:
I am not sure if adding `this VTGate` helps. 
Any time-series database will accumulate data from all VTGates and show a single view. 
However, users can add `instance` as a dimension to view it on a per-VTGate basis. 
The metrics themselves are collected at a single-process level.